specification/automation/resource-manager/Microsoft.Automation/stable/2015-10-31/definitions.json

```diff
@@ -2439,9 +2439,6 @@
           "description": "Gets or sets the tags attached to the resource."
         }
       },
-      "required": [
```

Comment:
This could be a breaking change. Please acknowledge.

Reply:
Please help me understand this. The existing clients expect "properties" to be required, so they will keep providing it, and the server will keep accepting it. This change removes a restriction, does not introduce a new one, so it should not be breaking. Am I missing anything?

Reply:
The client methods that Autorest generates has signatures where the required parameters appear first while the optional parameters with default parameters appear towards the end of the parameter list. Changing a required parameter to a non required can shift this parameter ordering for the methods and hence break users who have written code to use the method with the old parameter ordering.

Reply:
@dsgouda - Currently we (Automation team) are in the process of releasing the Autorest generated client to the public and do not have any external users. Also, our Autorest-SDKs are in [preview mode](https://www.nuget.org/packages/Microsoft.Azure.Management.Automation/3.0.0-preview) currently to help us fix such issues. Thus, this change should not be a problem. 
cc @AnatoliB

Reply:
Thanks for acknowledging this.